### PR TITLE
[NMA-878] Explore for wallet with Merchants/ATM's

### PR DESCRIPTION
**Pull request by Syn-McJ**
## Issue being fixed or feature implemented
Explore screen for Dash Wallet with Merchants/ATM option. The Explore Dash feature is moved to a separate feature module and is using navigation component.

## Screenshots / Videos
https://www.figma.com/file/84tUBQA9DYU1svyQx8tnbk/DashPay-%E2%80%93-Design-(Android)?node-id=1611%3A6352

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [x] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: features/exploredash/src/main/res/values/strings-explore-dash.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="explore_dash">Explore Dash</string>
+    <string name="explore_subtitle">Easily shop with your DASH at over 155,000 locations and online merchants</string>
+    <string name="explore_merchants_title">Where to Spend?</string>
+    <string name="explore_merchants_subtitle">Find merchants who accept Dash as payment.</string>
+    <string name="explore_atms_title">ATMs</string>
+    <string name="explore_atms_subtitle">Find where to buy or sell DASH and other cryptocurrencies for cash.</string>
+</resources>
```
Comment (HashEngineering) [inline features/exploredash/src/main/res/values/strings-explore-dash.xml:9]: new strings will require a change to our transifex config

make a modification to `.tx/config` to add a section for explore-dash

Here is an example from the liquid integration.  File name references are relative to the root folder of this repo:
```
[dash-mobile-wallets.strings-liquid]
file_filter = liquid-integration/src/main/res/values-<lang>/strings-liquid.xml
lang_map = sl_SI:sl, zh_TW:zh-rTW
source_file = liquid-integration/src/main/res/values/strings-liquid.xml
source_lang = en
type = ANDROID
```

**Review by HashEngineering — CHANGES_REQUESTED** [review]
One change is requested to support Transifex for this new module's string resources.

File: features/exploredash/src/main/res/values/strings-explore-dash.xml (hunk 1 above)
Comment (Syn-McJ) [reply features/exploredash/src/main/res/values/strings-explore-dash.xml:9]: Done

**Review by HashEngineering — APPROVED** [review]
Looks great.

**Pull request merged**